sdk/forms/src/options.ts

```diff
@@ -44,3 +44,9 @@ export type Hooks<T extends ResultType> = ErrorHook & {
 export type ErrorHook = {
   onError?: (error: Error) => void;
 };
+
+export type ProtectedViewHooks = ErrorHook & {
+  onClick?: (payload: unknown) => void;
+  onMouseEnter?: (payload: unknown) => void;
+  onMouseLeave?: (payload: unknown) => void;
```

Comment:
Add types for the payload.

Reply:
This is not an event, it's just a custom payload. I've tried to pass the entire event, but in that case it fails to execute postMessage because it can't make a copy of the entire Event object. That's why I've ended up passing only x and y from the event + property path, omitting everything else

Reply:
But I'll make typing more specific to reflect the structure of the payload

Reply:
Also updated README

Reply:
Yeah, I know it's not the actual original event object but still I think calling it event will be more intuitive for the user.

Reply:
Done, replaced with `event`